presto-orc/src/main/java/com/facebook/presto/orc/writer/StreamLayout.java

```diff
@@ -32,6 +33,10 @@
 {
     void reorder(List<StreamDataOutput> dataStreams);
 
+    void reorder(List<StreamDataOutput> dataStreams,
```

Comment:
should we remove the overload for other reorder ?

Reply:
I have removed ` void reorder(List<StreamDataOutput> dataStreams);` from the interface and for now passing immutable empty maps as input.